Incorrect documentation around password policies (#19364)
keycloak/keycloak: docs/documentation/server_admin/topics/authentication/password-policies.adoc

Password policy
image:images/password-policy.png[Password Policy]

After saving the policy, {project_name} enforces the policy for new users.

[NOTE]
====

> The new policy will not be effective for existing users. So, Make sure that you set the password policy from the beginning of the realm creation and/or add "Update password" to existing users and/or use "Expire password" to make sure that users update their passwords in next "N" days, which will in fact adjust to new password policies.

Reviewer: ```suggestion
The new policy will not be effective for existing users. Therefore, make sure that you set the password policy from the beginning of the realm creation or add "Update password" to existing users  or use "Expire password" to make sure that users update their passwords in next "N" days, which will actually adjust to new password policies.
```
